packages/celotool/src/cmds/bots/auto-vote.ts

```diff
@@ -19,21 +19,35 @@ export const describe = 'for each of the voting bot accounts, vote for the best
 
 interface SimulateVotingArgv {
   celoProvider: string
+  excludedGroups?: string[]
 }
 
 export const builder = (yargs: Argv) => {
-  return yargs.option('celoProvider', {
-    type: 'string',
-    description: 'The node to use',
-    default: 'http://localhost:8545',
-  })
+  return yargs
+    .option('celoProvider', {
+      type: 'string',
+      description: 'The node to use',
+      default: 'http://localhost:8545',
+    })
+    .option('excludedGroups', {
+      type: 'string',
+      description: 'A comma separated list of groups to exclude from voting eligibility',
+      coerce: (addresses: string) => {
+        return addresses
+          .split(',')
+          .filter((a) => a.length > 0)
+          .map(ensure0x)
```

Comment:
Instead of `ensure0x`, consider something like '0x' + `normalizeAddress` from @celo/utils/lib/address, here and elsewhere.  Any mapping from an address, e.g.`excludedGroups`, should normalize the keys.  This normalizes for '0x' prefix but I don't think for capitalization.

Reply:
oh, good catch! Thank you!